test/integration/parachain/staging/setup/initialize.test.ts

```diff
@@ -103,8 +98,7 @@ describe("Initialize parachain state", () => {
         userInterBtcAPI = new DefaultInterBtcApi(api, "regtest", userAccount, ESPLORA_BASE_PATH);
         sudoInterBtcAPI = new DefaultInterBtcApi(api, "regtest", sudoAccount, ESPLORA_BASE_PATH);
 
-        wrappedCurrency = userInterBtcAPI.getWrappedCurrency();
-        collateralCurrency = getCorrespondingCollateralCurrency(userInterBtcAPI.getGovernanceCurrency());
+        collateralCurrency = getCorrespondingCollateralCurrencies(userInterBtcAPI.getGovernanceCurrency())[0];
```

Comment:
Does this implicitly assume `KSM`? What happens to the tests if I change the return value of `getCorrespondingCollateralCurrency`?

Reply:
It only needs one collateral currency, whether that is KSM, or KINT doesn't matter here.
The currency is used in conjunction with vault account ids to wait for them to be available - as means to delay test execution until the parachain / docker images are good to go with vaults registered.